awot/io/flight.py

```diff
@@ -274,7 +277,7 @@ def read_nasa_ames(filename, mapping_dict=None, platform=None):
             name_map = _get_name_map(platform)
         else:
             name_map = _make_name_map_from_varlist(hdr['VNAME'])
-
+    print(name_map)
```

Comment:
Any chance you would put an optional variable ```verbose``` in the ```read_nasa_ames``` function and have an optional printout of variable names?